nodestream/schema/migrations/operations.py

```diff
@@ -179,6 +245,18 @@ def proposed_index_name(self) -> str:
     def describe(self) -> str:
         return f"Drop node type {self.name}"
 
+    def reduce(self, other: Operation) -> List[Operation]:
+        if isinstance(other, CreateNodeType) and other.name == self.name:
+            return []
+
+        if (
+            isinstance(other, (AddNodeProperty, DropNodeProperty, RenameNodeProperty))
+            and other.node_type == self.name
+        ):
+            return [self]
+
+        return Operation.reduce(self, other)
```

Comment:
If we make a new Operation like AddNodeLabel and we forget to add it into the list of operations to check for, we can have a situation like the following:

operations = [AddNodeLabel, DropNodeType]
resulting_operations = [DropNodeType, AddNodeLabel] = Operation.reduce(DropNodeType (self), AddNodeLabel (other))
Which would cause an error. 

I think this piece should return Operation.reduce(other, self) so that it maintains the original order of operations 
given that a reduction comparison is not given. 

[CreateNodeType, AddNodeProperty, AddNodeLabel, DropNodeType]
iterationA:
[AddNodeProperty, AddNodeLabel, DropNodeType]
Iteration B:
[AddNodeLabel, DropNodeType]
Iteration C:
[DropNodeType, AddNodeLabel]

Reply:
Also this prevents infinite loops I noticed Iteration C above would swap indefinately if AddNodeLabel does the same thing with Operation.reduce(AddNodeLabel, DropNodeType) -> [AddNodeLabel, DropNodeType] which is != [DropNodeType, AddNodeLabel]